gradle/geode-dependency-management.gradle

```diff
@@ -18,146 +18,245 @@
-    dependency('org.seleniumhq.selenium:selenium-chrome-driver:3.13.0')
-    dependency('org.seleniumhq.selenium:selenium-remote-driver:3.13.0')
-    dependency('org.seleniumhq.selenium:selenium-support:3.13.0')
-    dependency('org.skyscreamer:jsonassert:1.5.0')
-    dependency('org.slf4j:slf4j-api:1.7.25')
-    dependency('org.springframework.hateoas:spring-hateoas:0.25.0.RELEASE')
-    dependency('org.springframework.ldap:spring-ldap-core:2.3.2.RELEASE')
-    dependency('org.springframework.security:spring-security-config:4.2.8.RELEASE')
-    dependency('org.springframework.security:spring-security-core:4.2.8.RELEASE')
-    dependency('org.springframework.security:spring-security-ldap:4.2.8.RELEASE')
-    dependency('org.springframework.security:spring-security-test:4.2.8.RELEASE')
-    dependency('org.springframework.security:spring-security-web:4.2.8.RELEASE')
-    dependency('org.springframework.shell:spring-shell:1.2.0.RELEASE')
-    dependency('org.springframework:spring-aspects:4.3.20.RELEASE')
-    dependency('org.springframework:spring-beans:4.3.20.RELEASE')
-    dependency('org.springframework:spring-context:4.3.20.RELEASE')
-    dependency('org.springframework:spring-core:4.3.20.RELEASE')
-    dependency('org.springframework:spring-expression:4.3.20.RELEASE')
-    dependency('org.springframework:spring-oxm:4.3.20.RELEASE')
-    dependency('org.springframework:spring-test:4.3.20.RELEASE')
-    dependency('org.springframework:spring-tx:4.3.20.RELEASE')
-    dependency('org.springframework:spring-web:4.3.20.RELEASE')
-    dependency('org.springframework:spring-webmvc:4.3.20.RELEASE')
-    dependency('pl.pragmatists:JUnitParams:1.1.0')
-    dependency('redis.clients:jedis:2.9.0')
-    dependency('xerces:xercesImpl:2.12.0')
+    project.logger.info("Applying external property version values via GeodeDependencyManagementPlugin to ${project}")
+    set_extended_property_versioning(project)
+  }
+
+  void manage_dependencies(Project project) {
+    project.apply plugin: "io.spring.dependency-management"
+    project.dependencyManagement {
+      dependencies {
+        dependencySet(group: project.group, version: project.version) {
+          entry('apache-geode')
+          entry('geode-common')
+          entry('geode-concurrency-test')
+          entry('geode-connectors')
+          entry('geode-core')
+          entry('geode-cq')
+          entry('geode-dunit')
+          entry('geode-experimental-driver')
+          entry('geode-json')
+          entry('geode-junit')
+          entry('geode-lucene')
+          entry('geode-old-client-support')
+          entry('geode-old-versions')
+          entry('geode-protobuf')
+          entry('geode-protobuf-messages')
+          entry('geode-pulse')
+          entry('geode-rebalancer')
+          entry('geode-wan')
+          entry('geode-web')
+          entry('geode-web-api')
+        }
+
+        // informal, inter-group dependencySet
+        dependency('com.sun.xml.bind:jaxb-impl:2.3.1')
```

Comment:
Do we know if these versions are actually coupled?

Reply:
Dunno!  I just noticed they're both JAXB and on the same version.  I'll dig into it.

Reply:
As best as I can tell, this is just pathological out in Maven.

[jaxb-impl](https://mvnrepository.com/artifact/com.sun.xml.bind/jaxb-impl/2.3.1) from Sun is the up-to-date version of the impl, but [jaxb-api](https://mvnrepository.com/artifact/javax.xml.bind/jaxb-api/2.3.1) from jaxb.xml.bind is the up-to-date api.  Everything else similarly named appears to have been abandoned in 2010.